package.json

```
@@ -111,6 +111,6 @@
   "engines": {
     "node": ">=12.0.0",
     "npm": ">=6.0.0",
-    "bee": "0.5.3-acbd0e2"
+    "bee": "0.6.0-acbd0e2"
```

```suggestion
    "bee": "0.6.0-c1e87ce-dirty"
```
Don't forget to change the commit hash when Bee is actually released.

---

This is already part of #307